department-of-veterans-affairs/caseflow
# Correspondence and Appeal Tasks: Inbound Ops Team Superuser/visor: Remove Evidence Submission Window Action Dropdown

Resolves [Correspondence and Appeal Tasks: Inbound Ops Team Superuser/visor: Remove Evidence Submission Window Action Dropdown](https://jira.devops.va.gov/browse/APPEALS-58227)

# Description
This PR introduces the ability to view completed evidence submission tasks within the correspondence details component. The page also introduces a dropdown that allows the user to remove the waive evidence window period. This dropdown is only visible if the window period is currently less than 90 days, the user is a superuser or supervisor of the inbound ops team, and the task is completed.

## Acceptance Criteria

## Testing Plan
1. Take a correspondence through intake and make sure to add at least one task related to an appeal to it. 
2. Visit the correspondence details for the correspondence and find the evidence submission window task listed. Note that it does not have the dropdown in its current status.
3. Set the evidence submission task into a completed state.
4. Swap users to a supervisor or superuser for inbound ops team.
5. Note that the dropdown now displays.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
<img width="1179" alt="image" src="https://github.com/user-attachments/assets/58e7e4db-c4fd-4f20-acc6-5b0bfc6c5ae7">
|
<img width="1270" alt="image" src="https://github.com/user-attachments/assets/a0545485-9caf-4dc9-9697-dbe67f36c62a">
